src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -4,7 +4,7 @@ Class {
 	#category : #'Math-Tests-Complex'
 }
 
-{ #category : #tests }
+{ #category : #'unary operations' }
```

Comment:
I would prefer 'tests - unary operations'. Some library use this pattern.

Reply:
Cool. Is this something you'd like for the other categories ?

Reply:
I'll make the change in the next commit.